packages/core/src/bundle.ts

```diff
@@ -271,7 +272,8 @@ function makeBundleVisitor(version: OasMajorVersion, dereference: boolean, rootD
       }
     }
 
-    name = refBaseName(fileRef) + (name ? `_${name}` : '');
+    const separator = isAbsoluteUrl(fileRef) ? '/' : platformDependentSeparator;
```

Comment:
Better to not use `platformDependentSeparator`... We should expect all paths in yaml file to have linux styled separators. Otherwise it won't work on our workflows runner.

Linux slashes work on windows too but not other way.

Reply:
In this case we're overriding separator only when absolute url is met. It won't affect relative references. Or I'm missing something ?

Reply:
Oh, I see it was using path.sep before... I think this is a mistake. We should use `/[\/\\]/` regexp in windows, windows accepts forward slashes in most of its API at this point: both `C:\test/a/b/c` and `C:\test\a\b\c` are valid in Windows.

Reply:
`/[\/\\]/` works good on mac. let me test it on windows

Reply:
Works good on Windows as well 🎉